File: ConverterToolTests/SolutionFiles/JavaFiles/AbstractMethods.java
import java.io.*;
import java.lang.*;
import java.util.*;

public abstract class AbstractMethods
{
    private int _x;
    private int _y;
    void moveTo(int newX, int newY) {
        this._x = newX;
        this._y = newY;
    }
    abstract void draw();
    abstract void resize();
}

File: ConverterToolTests/SolutionFiles/CsharpFiles/AbstractMethods.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ConverterToolTests.SolutionFiles.CsharpFiles
{
    public abstract class AbstractMethods
    {
        private int _x = 0;
        private int _y = 0;
        void moveTo(int newX, int newY)
        {
            this._x = newX;
            this._y = newY;
        }
        public abstract void Draw();
        public abstract void Resize();
    }
}
